thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -553,6 +546,72 @@ private String getDimensions(String dataset, String exploreDimensions) throws Ex
     return reorderedExploreDimensions.toString();
   }
 
+  /**
+   * Show the content of merged anomalies whose start time is located in the given time ranges
+   *
+   * @param functionId id of the anomaly function
+   * @param startTimeIso The start time of the monitoring window
+   * @param endTimeIso The start time of the monitoring window
+   * @param anomalyType type of anomaly: "raw" or "merged"
+   * @param applyAlertFiler can choose apply alert filter when query merged anomaly results
+   * @return list of anomalyIds (Long)
+   */
+  @GET
+  @Path("anomaly-function/{id}/anomalies")
+  public List<Long> getAnomaliesByFunctionId(@PathParam("id") Long functionId,
```

Comment:
1. This resource file is written for version 1 dashboard. The latest resource file is AnomaliesResource.java. Could you move this method there? There is a block in which the methods are named as getAnomaliesBySomething.

2. Rename to getAnomalyIdsByFunctionId.